Release alarm window when alarm metrics is expired in case OOM (#11336)
apache/skywalking: docs/en/changes/changes.md

* Support Kafka Monitoring.
* [Breaking Change] Elasticsearch storage merge all management data indices into one index `management`, 
  including `ui_template，ui_menu，continuous_profiling_policy`.

> * Fix Release alarm window when alarm metric is expired in case OOM.

Reviewer: ```suggestion
* Add a release mechanism for alarm windows when it is expired in case of OOM.
```

Fix words.
